pkg/analysis_server/lib/src/computer/computer_closingLabels.dart

```diff
@@ -7,13 +7,22 @@ import 'package:analyzer/dart/ast/ast.dart';
 import 'package:analyzer/dart/ast/visitor.dart';
 import 'package:analyzer/src/generated/source.dart';
 
+class _ClosingLabelWithLineCount {
+  final ClosingLabel label;
+  final int spannedLines;
+
+  _ClosingLabelWithLineCount(ClosingLabel label, int spannedLines)
+      : this.label = label,
+        this.spannedLines = spannedLines;
+}
```

Comment:
Thanks! I actually tried something like this (probably using typescript syntax, where you don't need to declare the fields if you put public/private in front of the constructor arg) but it failed. I should've looked it up!